components/org.wso2.carbon.identity.scim.provider/src/main/webapp/WEB-INF/cxf-servlet.xml

```diff
@@ -46,6 +46,11 @@
         <jaxrs:providers>
             <ref bean="authenticationFilter"/>
         </jaxrs:providers>
+
+        <jaxrs:properties>
+            <entry key="org.apache.cxf.jaxrs.allow.http.method.override" value="true"/>
+        </jaxrs:properties>
```

Comment:
With CXF 3.2.11 onwards HTTP method overriding is disabled by default [1]. Therefore [2] will fail when upgrading CXF to 3.3.7 due to HTTP method overriding at [3]. Adding this config will resolve the test failure.

[1] https://issues.apache.org/jira/browse/CXF-8133
[2] https://github.com/wso2/product-is/blob/master/modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderGroupTestCase.java#L218
[3] https://github.com/wso2/product-is/blob/master/modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim/SCIMServiceProviderGroupTestCase.java#L248